cblite/CdCommand.cc

```diff
@@ -36,10 +36,9 @@ class CdCommand : public CBLiteCommand {
 
 
     void usage() override {
-        writeUsageCommand("cd", true, "[COLLECTION]");
+        writeUsageCommand("cd", true, "[COLLECTION] [SCOPE]");
```

Comment:
This parameter order feels unintuitive to me. Since scopes contain collections, I'd expect the scope name to come first. So it would be `cd COLLECTION` or `cd SCOPE COLLECTION`. What do you think?

Alternatively, the way I was imagining adding scope support was to use a slash as a delimiter, so you'd say `cd SCOPE/COLLECTION`.

Reply:
The reason I had it like this is basically because scope has a sort of "default argument" but collection is always required.  The second form could handle that though.